fortuna/public/index.html

```diff
@@ -1,43 +1,43 @@
-<!DOCTYPE html>
```

Comment:
Did something happen that changed this entire file? If nothing did, there shouldn't be a need to include this, if possible.

Reply:
This is in public, not src, so it is autogenerated and shouldn't matter